metro.config.js

```diff
@@ -7,8 +8,27 @@ const gutenbergMetroConfigCopy = {
 gutenbergMetroConfigCopy.resolver.extraNodeModules = new Proxy(
 	{},
 	{
-		get: ( target, name ) =>
-			path.join( process.cwd(), `gutenberg/node_modules/${ name }` ),
+		get: ( target, name ) => {
+			const gutenbergFolder = path.join(
+				process.cwd(),
+				`gutenberg/node_modules/${ name }`
+			);
+			if ( fs.existsSync( gutenbergFolder ) ) {
+				return gutenbergFolder;
+			}
+
+			// let's try find the module in the Jetpack submodule. We'll try the .pnpm folder.
+			const moduleFolderPnpm = path.join(
+				process.cwd(),
+				`./jetpack/node_modules/.pnpm/node_modules/${ name }`
+			);
+
+			// pnpm uses symlinks so, let's find the target
+			const symlinkTarget = fs.readlinkSync( moduleFolderPnpm );
+
+			// the target is still using paths relative to the parent folder of the module, let's find the real path.
+			return path.resolve( moduleFolderPnpm + '/../' + symlinkTarget );
```

Comment:
>Had to [add more logic to the Metro package resolver](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3575/files#diff-eb288b08cb881cff653283c14cb9630dc0924a1751950e1562a8ca2c0b39694bR20-R33) (and thank you @ceyhun for the guidance on how we use the Proxy there!) to help Metro find the Jetpack package dependencies as installed by pnpm. Happy to find a simpler way to point Metro to the right place if there is one!

For better or worse, I do believe this is likely as "simple" as it gets until [Metro supports symlinks](https://github.com/facebook/metro/issues/1).

Reply:
Cool, thanks for adding your "stamp" on it David! 🙇✅